dev/experimental.agda-lib

```diff
@@ -0,0 +1,4 @@
+name: standard-library-dev
+include: . ../src
+flags:
+  --warning=noUnsupportedIndexedMatch
```

Comment:
This makes perfect sense as an incarnation of the orthodox library `import` convention.

But for end users/new stdlib developers, what, if anything, should they add to their `~/.agda/libraries` to pick up the `development` library rather than the `release-version` library during their own development? Obviously, this can't be part of the PR, but can it be documented in `HACKING.md` for example?

Reply:
Isn't that just the standard agda docs about managing libraries?
There's nothing special about the dev version.